pom.xml

```diff
@@ -400,6 +400,16 @@
       <artifactId>checker-qual</artifactId>
       <version>${checkerframework.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.maven.doxia</groupId>
+      <artifactId>doxia-core</artifactId>
+      <version>1.12.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.maven.doxia</groupId>
+      <artifactId>doxia-module-xdoc</artifactId>
+      <version>1.12.0</version>
```

Comment:
Let's make property for version, to update it by one line change in future

Reply:
Done. Created `doxia.version`